brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3207 Resolved Customizer Preview issue in FAQ, IconList, SocialShare, PriceList Parent.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3207

### Screenshots

- https://d.pr/v/I3QskM

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Go Astra Customizer > Header > Widget
- Use Icon List, FAQ etc ( Use block which has parent child relationship )
- You will see its broken when you are on master branch.
- Now switch to branch and refresh the customizer do the same step issue will get resolve.
- Try adding to same block again and customize the style of the block.

### Checklist:
